src/components/Tabs/Tabs.tsx

```diff
@@ -43,7 +43,7 @@ export const Tabs: React.FC<TabsProps> = ({
     size = 'm',
     activeTab,
     allowNotSelected = false,
-    items = [],
+    items,
```

Comment:
Looks like a breaking change

Reply:
This prop is required by types https://github.com/yandex-cloud/uikit/blob/main/src/components/Tabs/Tabs.tsx#L32
Maybe we should make it optional then?